src/components/state.tsx

```diff
@@ -0,0 +1,31 @@
+import { createContext, ReactNode, useState } from 'react';
+import { Person } from '../types';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const MyContext = createContext<any>(null);
+
+interface MyProvideProps {
+  children: ReactNode;
+}
+
+export const MyProvider = ({ children }: MyProvideProps) => {
+  const [person, setPerson] = useState<Person[]>([]);
+  const [error, setError] = useState('');
+  const [isLoading, setIsLoading] = useState<boolean>(false);
```

Comment:
The initial state for `isLoading` is redundant since you're already specifying the type in the hook. You can simply use `useState(false)`.